ports/stm32/boards/WEACT_F411_BLACKPILL/stm32f4xx_hal_conf.h

```diff
@@ -0,0 +1,23 @@
+/* This file is part of the MicroPython project, http://micropython.org/
+ * The MIT License (MIT)
+ * Copyright (c) 2024 Andrew Leech
+ */
+#ifndef MICROPY_INCLUDED_STM32F4XX_HAL_CONF_H
+#define MICROPY_INCLUDED_STM32F4XX_HAL_CONF_H
+
+#include "boards/stm32f4xx_hal_conf_base.h"
+
+// Oscillator values in Hz
+#if MICROPY_HW_HSE_VALUE
```

Comment:
In this case I think it's better to do `#ifdef MICROPY_HW_HSE_VALUE`, because if that value is defined it should be used (even if it's 0...).